### PR TITLE
add 4.12 job for e2e-aws-ocm-osd-ccs-p1

### PR DESCRIPTION
**Pull request by yasun1**

### CODE REVIEW

**Comment by kuiwang02** [comment]
I am good.
I think you need qe productivity team to review it.

**Comment by yasun1** [comment]
Thanks Kui. 
/assign @liangxia please help review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -1203,6 +1203,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-proxy
+- as: e2e-aws-ocm-osd-ccs-p1
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:1206]: Please sort the jobs on this line.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 2]
```diff
@@ -1203,6 +1203,19 @@ tests:
...
+  cron: 0 16 * * 3
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      E2E_RUN_TAGS: '@aws-ipi and @osd_ccs and @network-openshiftsdn and not @fips'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:1212]: `network-openshiftsdn` should be `network-ovnkubernetes`

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml (hunk 1 above)
Comment (yasun1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml]: Strangely, the 'make update' does not help me do the sorting automatically. I will do it manually.

**Pull request merged**